sdk/deepbook/src/deepbook_query.ts

```diff
@@ -26,21 +14,21 @@ export class DeepBook_query {
 	 * @param token1 token1 of a certain pair, eg: 0x5378a0e7495723f7d942366a125a6556cf56f573fa2bb7171b554a2986c4229a::weth::WETH
 	 * @param token2 token2 of a certain pair, eg: 0x5378a0e7495723f7d942366a125a6556cf56f573fa2bb7171b554a2986c4229a::usdt::USDT
 	 * @param poolId: the pool id, eg: 0xcaee8e1c046b58e55196105f1436a2337dcaa0c340a7a8c8baf65e4afb8823a4
-	 * @param orderId the order id, eg: 1
+	 * @param orderId the order id, eg: "1"
 	 * @param accountCap: your accountCap, eg: 0x6f699fef193723277559c8f499ca3706121a65ac96d273151b8e52deb29135d3
 	 */
 	public async get_order_status(
 		token1: string,
 		token2: string,
 		poolId: string,
-		orderId: number,
+		orderId: string,
 		accountCap: string,
 	): Promise<DevInspectResults> {
 		const txb = new TransactionBlock();
 		txb.moveCall({
 			typeArguments: [token1, token2],
-			target: `dee9::clob::get_order_status`,
-			arguments: [txb.object(poolId), txb.object(String(orderId)), txb.object(accountCap)],
+			target: `dee9::clob_v2::get_order_status`,
+			arguments: [txb.object(`${poolId}`), txb.object(String(orderId)), txb.object(`${accountCap}`)],
```

Comment:
You shouldn't need to wrap this in a template string, these all already look like strings

Reply:
remove all the unneeded wraps